views/movies/new-movie.hbs

```diff
@@ -0,0 +1,21 @@
+<h2>Create movie</h2>
+<form action="/movie/create" method="post">
```

Comment:
Here on the form when the user clicks the create button you are redirecting the user to `"/movie/create"` but the post route that you have created is `"/movies/create"` that's why it's giving you the status code 404